src/main/java/io/hops/metadata/hdfs/entity/RetryCacheEntry.java

```diff
@@ -18,48 +18,58 @@
 package io.hops.metadata.hdfs.entity;
 
 import io.hops.metadata.common.FinderType;
-import java.util.Arrays;
+import org.omg.PortableInterceptor.SUCCESSFUL;
 
+import java.util.Arrays;
+import java.util.Objects;
 
 public class RetryCacheEntry {
+
+  public static byte SUCCESS = (byte) 1;
+  public static byte FAILED = (byte) -1;
```

Comment:
Is this used? Shouldn't it match or replace: https://github.com/hopshadoop/hops/blob/d2f89602e4a3b92501f392de1a40a7cf514311b4/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/RetryCache.java#L60

Reply:
Its  fine. We have our own implementation so Values can differ